services/web/client/source/class/osparc/component/workbench/NodeUI.js

```diff
@@ -107,44 +104,56 @@ qx.Class.define("osparc.component.workbench.NodeUI", {
       return this.getNode().getMetaData();
     },
 
-    __createNodeLayout: function() {
-      this.setLayout(new qx.ui.layout.VBox());
+    _createChildControlImpl: function(id) {
```

Comment:
the implementation depends on the call order ... this is not ideal ... by using a `grid` layout this problem could be avoided by inserting the individual elements into respective 'rows'.